### DISCUSSION TITLE
intestinal intrinsic clearance

### DISCUSSION THREAD

**Discussion by Carrie-nothing** [Open-Systems-Pharmacology/Forum#1136] [Questions & Problems]
Hello,
Teams.

I would like to add an intestinal CYP3A biotransformation.
Firstly, I add the concentration of binding/metabolizing protein;
Secondly, I make the steps shown in the figure.
![image](https://user-images.githubusercontent.com/92715703/154499374-947724dd-8c53-4fe6-a9a0-ad609f892e0a.png)
As for the blue parts, that is a liver content, but I need to make an intestinal content.
Should I need to use Mobi?
Can you help me to solve this problem?
Thank you very much!

Yours,
Carrie

**Comment by prvmalik** [comment]
First, please be aware that for metabolism the only parameter that is used in PKSim is the 'specific clearance' for first order processes and 'kcat' for Michaelis-Menten processes. The calculation parameters do not matter. The calculation parameters are just rough guesses or conversions for arriving at 'specific clearance' from an intrinsic clearance value. 

So what you could then is instead use the 'in vitro clearance - first order' formula rather than the 'intrinsic clearance - first order' formula for your gut metabolism because in this sense you define specific clearance directly and cleanly. Provide some justification for how you convert from intrinsic clearance to specific clearance if you would like to use an intrinsic clearance value from enterocytes but this is generally not well-scalable and requires optimization. Specific clearance is the actual first order rate of metabolism inside cells.

When doing gut metabolism you'll want to make sure that the gut metabolizing enzyme is expressed intracellularly in mucosal cells and you may or may not want it in the normal intestinal cells (i.e. do you want it contributing to systemic metabolism or just first pass?)

The most ideal way of doing this would be by adding a CYP3A4 enzyme (liver, gut, whole body) with relative expression and being able to modulate the degree of intestinal metabolism with a scaling factor as I have suggested before. [https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1897]

Other ways that gut metabolism has been implemented is by Sebastian Frechen by modulating the basolateral intestinal permeability (see midazolam, finerenone, alfentanil) to increase residence time in the mucosa, but this does not scale well to special populations or for certain DDIs. In SimCyp what is done is increasing the fraction unbound in gut mucosa.

**Reply by Carrie-nothing** [reply]
Thank you for the reply!
I imitated a paper using PKsim, which added the amount of intestinal enzymes and the way of intestinal intrinsic clearance in PKsim.
Its title is 'Comparing Predictions of a PBPK Model for Cyclosporine With Drug Levels From Therapeutic Drug Monitoring' 
And the contents about the Intestinal CYP3A4 biotransformation:
'The kinetics of systemic biotransformation were again quantified based on Tanaka et al. While this model was developed for intravenous application and supposed biotransformation in the liver only, a closer agreement with observed data could be achieved with the inclusion of biotransformation in more CYP3A4-enriched organs as mentioned above. The abundance and relative distribution of CYP 3A4 was quantified based on in vitro data as specified in Table 3.
![image](https://user-images.githubusercontent.com/92715703/154622131-f45029c8-6253-4e71-aab1-969b6a9da8aa.png)
To reproduce the bioavailability of cyclosporine after oral intake the intestinal biotransformation during absorption was quantified excluding systemic metabolism by fitting the intrinsic intestinal clearance to reach a literature based value for FI.'

So if I use the 'in vitro clearance - first order' formula ,I have directly defined 'Specific clearance' rather than the 'intrinsic clearance' mentioned in the article.Am I right in this conclusion?

Thanks again for your reply!

**Reply by Carrie-nothing** [reply]
About  the scaling factor.
I can't add a scalar to Vmax or Specific clearance of the relevant CYP process in the intracellular mucosa compartment of the small intestine segments.
How can I make it?
Thank you very much!

**Reply by NataliaRivaSJ** [reply]
@Carrie-nothing How did you solve this task? thank you in advance for your consideration,
best regards
